larray/viewer/model.py

```diff
@@ -227,16 +226,28 @@ def _set_data(self, data, xlabels, ylabels, changes=None, bg_gradient=None, bg_v
 
         assert isinstance(changes, dict)
         self.changes = changes
-        self._data = data
-        if xlabels is None:
-            xlabels = [[], []]
-        self.xlabels = xlabels
-        if ylabels is None:
-            ylabels = [[]]
-        self.ylabels = ylabels
-
-        self.total_rows = self._data.shape[0]
-        self.total_cols = self._data.shape[1]
+
+        if la_data.ndim == 0:
+            axes = la.AxisCollection((1, 1))
```

Comment:
modifying axes for 0d and 1d arrays like this means they will display exactly like 2d arrays, which will be ugly and confusing, especially because of the extra filter button(s). This wasn't the case previously.